riscv-executor/src/lib.rs

```diff
@@ -626,6 +626,10 @@ mod builder {
             }
         }
 
+        pub fn col_is_defined(&self, name: &str) -> bool {
```

Comment:
can you make this a `try_get_col` which return an Option instead? And checks that its in ExecMode::Trace? (see `get_col`)